CHANGELOG.md

```diff
@@ -10,12 +10,15 @@ All notable changes to TTT2 will be documented here. Inspired by [keep a changel
   - introduced new development interfaces to easily create menues and settings for addons
 - Introduced a global scale factor based on screen resolution to scale HUD elements accordingly
 - Added automatical scale factor change on resolution change that works even if the resolution was changed while TTT2 wasn't loaded
+- Added scale based draw functions
```

Comment:
i think it should be pointed out that this is a library, so it can be found more easily in the codebase.
something along the lines of:
```suggestion
- Added `drawsc` library featuring scalable draw functions
```